src/main/java/randoop/contract/CompareToEquals.java

```diff
@@ -10,10 +10,13 @@
  * <pre>(x0.compareTo(x1) == 0) == x0.equals(x1)</pre>
  */
 public class CompareToEquals extends ObjectContract {
+  /** The singleton instance of this class. */
   private static final CompareToEquals instance = new CompareToEquals();
 
-  private CompareToEquals() {}
+  /** Singleton pattern. */
```

Comment:
The documentation should be something like "Creates the singleton CompareToEquals; is only ever called once.  Clients should call {@link #getInstance}."